frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -523,6 +524,11 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                             <UsageTab id={id} featureFlag={featureFlag} />
                                         </Tabs.TabPane>
                                     )}
+                                    {featureFlag.key && id && (
+                                        <Tabs.TabPane tab={<div>Analysis</div>} key="analysis">
```

Comment:
when you feature flag it, can you also use the `PostHogFeature` component?

And also enable beta access?

Reply:
Going to follow up after this: https://github.com/PostHog/posthog/pull/16005